service/src/main/kotlin/app/cash/backfila/ui/actions/BackfillCreateHandlerAction.kt

```diff
@@ -48,8 +51,39 @@ class BackfillCreateHandlerAction @Inject constructor(
           else -> true
         },
       )
-      formFields[BackfillCreateField.RANGE_START.fieldId]?.ifNotBlank { createRequestBuilder.pkey_range_start(it.encodeUtf8()) }
-      formFields[BackfillCreateField.RANGE_END.fieldId]?.ifNotBlank { createRequestBuilder.pkey_range_end(it.encodeUtf8()) }
+
+      // Handle range options for cloned backfills
+      when (formFields[BackfillCreateField.RANGE_OPTION.fieldId]) {
+        RangeOption.CONTINUE.value -> {
+          // Get the last processed position from the original backfill
+          val backfillId = formFields[BackfillCreateField.BACKFILL_NAME.fieldId]?.toLongOrNull()
```

Comment:
This is going to be the fully qualified name of the backfill class, not the DB ID as a Long so this parse will always resolve to null. You'll need to add BACKFILL_ID_TO_CLONE as a new enum and hidden input field to fill in the create action so you can get it in the create handler.